topics/lua-api.md

```diff
@@ -502,9 +506,8 @@ You can use the following flags and instruct the server to treat the scripts' ex
 
 Please refer to [Function Flags](functions-intro.md#function-flags) and [Eval Flags](eval-intro.md#eval-flags) for a detailed example.
 
-### <a name="redis.redis_version"></a> `redis.REDIS_VERSION`
+### <a name="server.server_version"></a> `server.SERVER_VERISON`
 
-* Since version: 7.0.0
```

Comment:
I think the version information is important. I disagree with the PR deleting all this.

Users may write Lua scripts that are need to run on a range of versions of Valkey and Redis OSS, so depending on which versions they need to support, they need to know what they can use. Let's say they're writing some open source lib or tool or so.

For things that aren't yet released, it's even more important, but this one is new in Valkey 7.2.5, right?

Somewhere in the docs, we can write that versions < 7.2.5 always refer to Redis OSS and >= 7.2.5 refer to Valkey versions.

Reply:
I'm not sure how strongly I feel about that. There are some very old versions of Redis that have weird behavior, are we going to document all of that behavior? Is there a single issue where we are discussing this? Should we add it to the contributor meeting we have on Monday?

> Users may write Lua scripts that are need to run on a range of versions of Valkey and Redis OSS, so depending on which versions they need to support, they need to know what they can use. Let's say they're writing some open source lib or tool or so.

I don't think it's our job to detail all of the historical behavior changes in Redis. I am documenting all of the "breaking" changes, and intend to write a single page including everything that could break if you are moving from Redis to Valkey.

Reply:
Old versions, please delete. The only ones (IMO) that may be of interest are the still supported versions, i.e. 6.2, 7.0 and 7.2.

It's been discussed in random PRs. There's no single place AFAIK.

> I don't think it's our job to detail all of the historical behavior changes in Redis.

That's one way to look at it.

Here's another: If you consider Valkey the continuation of Redis OSS that just changed its name, then it does make sense to keep historical info. We're still the same project, so version history is as useless or useful as it would be without a name change.

> intend to write a single page including everything that could break if you are moving from Redis to Valkey.

Good, but I think that's release notes material. That's what people should review when updating. If people don't read the release notes, then maybe they're not accessible enough? We should link to them on the download page next to each version.

Reply:
> Old versions, please delete. The only ones (IMO) that may be of interest are the still supported versions, i.e. 6.2, 7.0 and 7.2.

Ok. I'd commit to keeping information about versions 7-6.2 for historical purposes. We can document we drop references from versions of Redis before that point.

> Good, but I think that's release notes material. That's what people should review when updating. If people don't read the release notes, then maybe they're not accessible enough? We should link to them on the download page next to each version.

I was going to do it as a one-off to build trust for folks to migrate. I agree generally that is for release notes, but it will be confusing initially as folks move from Redis -> valkey.